params/optimism_feature_toggles.go

```diff
@@ -0,0 +1,13 @@
+package params
+
+// OPStack diff
+// This file contains ephemeral feature toggles which should be removed
+// after the fork scope is locked.
+
+func (c *ChainConfig) IsMinBaseFee(time uint64) bool {
+	return c.IsJovian(time) // Replace with return false to disable
+}
+
+func (c *ChainConfig) IsDAFootprintBlockLimit(time uint64) bool {
```

Comment:
There's no impl work for this yet, but we can insert this now for in flight work to rebase onto.